src/content/docs/distribute/snapcraft.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Snapcraft
+sidebar:
+  order: 1
+---
+# Prerequisites
+
+import { Tabs, TabItem, Card } from '@astrojs/starlight/components';
+
+**1. Install snap** 
+<Tabs>
+  <TabItem label="Debian">
+```sh
+  sudo apt install snapd
+```
+  </TabItem>
+<TabItem label="Arch">
+```sh
+sudo pacman -S --needed git base-devel
+  git clone https://aur.archlinux.org/snapd.git
+  cd snapd
+  makepkg -si
+  sudo systemctl enable --now snapd.socket
+  sudo systemctl start snapd.socket
+  sudo systemctl enable --now snapd.apparmor.service
+```
+  </TabItem>
+<TabItem label="Fedora">
+```sh
+  sudo dnf install snapd
+  sudo ln -s /var/lib/snapd/snap /snap # Enables classic snap support
+  #Reboot your system after doing this
+```
+  </TabItem>
+</Tabs>
+**2. Installing Core**
+<Tabs>
+  <TabItem label="V1">
```

Comment:
I assume this means Tauri v1? We keep these docs v2 only so it's fine to remove this.

Reply:
Ok will do, I will make separate docs for v1

Reply:
If you'd like sure, but don't worry about it too much, at this point we don't care about v1 docs _that_ much anymore tbh (we're close to v2 release-candidate)